lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu
+
+	attr_reader :menu_list, :menu_keys, :menu_values
+
+	def initialize
+		@menu_list = menu_list
+		@menu_keys = menu_list.keys
+		@menu_values = menu_list.values
+	end
+
+		def menu_list
+			menu_list = {Burger: 6,
+							FishandChips: 7,
+							PieandMash: 9,
+							Pizza: 8,
+							Beer: 4,
+							Wine: 5,
+							Juice: 2}
+		end
+
+		def show
+			puts "MENU"
+			count = 1
+			menu_list.each do |key, value|
```

Comment:
Very nice! You could use `each_with_index` here to avoid using a `count`:

```
menu_list.each_with_index do |(key, value), index|
  p "#{index}: #{key} - £#{value}"
end
```